ui/perfherder/App.jsx

```diff
@@ -237,11 +236,6 @@ class App extends React.Component {
             </Switch>
           </main>
         )}
-        {errorMessages.length > 0 && (
-          <Container className="pt-5 max-width-default">
-            <ErrorMessages errorMessages={errorMessages} />
-          </Container>
-        )}
```

Comment:
You need to ensure the other routes in this component also have error handling, since a user can login anywhere within the Perfherder app. So GraphsView and CompareView and CompareSubtestViews need to have errormessages passed in as props.